src/main/java/org/hl7/davinci/priorauth/PopulateDatabaseEndpoint.java

```diff
@@ -0,0 +1,126 @@
+package org.hl7.davinci.priorauth;
+
+import org.hl7.davinci.priorauth.Database;
+import org.hl7.davinci.priorauth.FhirUtils;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.enterprise.context.RequestScoped;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriInfo;
+import javax.ws.rs.core.Response.Status;
+
+import org.hl7.fhir.r4.model.Bundle;
+import org.hl7.fhir.r4.model.Claim;
+import org.hl7.fhir.r4.model.ClaimResponse;
+
+import ca.uhn.fhir.context.FhirContext;
+
+@RequestScoped
+@Path("PopulateDatabase")
+public class PopulateDatabaseEndpoint {
```

Comment:
Should this only be enabled as a debug option?
Maybe the name should be something about PopulateTestData or something. I'm not sure I'm a fan of having this "test" code in with the "real" code.

Reply:
Yea that's not a bad idea. I can add a check on it to see if debug is enabled. Do you think this should be its own endpoint as it is now or should it be moved to the DebugEndpoint class (and we will probably want to update the root of the DebugEndpoint to be something like `debug` instead of `query`)

Reply:
I like the idea of combining it with the debug and changing query to debug.

Reply:
Updated so debug endpoint is now `debug` and the populate DB script runs on `PopulateDatabaseTestData` if debug is enabled